flink/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Check: flink
+
+## Overview
+
+This check monitors [Flink][1]. Datadog collects Flink metrics through Flink's
+[Datadog HTTP Reporter][4], which uses [Datadog's HTTP API][2].
+
+## Setup
+
+### Installation
+
+The Flink check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Configure the [Datadog HTTP Reporter][4] in Flink.
+
+    Copy `<FLINK_HOME>/opt/flink-metrics-datadog-1.9.2.jar` into your `<FLINK_HOME>/lib` folder.
+    In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `<API_KEY>` with your Datadog API key:
```

Comment:
```suggestion
    In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `<DATADOG_API_KEY>` with your Datadog API key:
```

Reply:
We should also add a link to: https://app.datadoghq.com/account/settings#api

Reply:
Updated!